_posts/2018-03-28-bedrock-the-sqlitening.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "Bedrock: The SQLitening"
+author: Paul McLanahan
+excerpt_separator: <!--more-->
+---
+
+On it's face [www.mozilla.org](https://www.mozilla.org/) doesn't look like it'd be a complex application to write, maintain, or run.
+But when you throw over 100 Million unique visitors per week at any site it can complicate things quickly. Add to that translations
```

Comment:
is `Million` capitalized on purpose?

Reply:
I think I started with 100M and changed it. Good catch.